webproject/src/main/webapp/index.html

```diff
@@ -225,23 +219,20 @@ <h2 class="section-title mb-3">Our Team</h2>
                   alt="Image"
                   class="w-100 img-fluid mb-3 shadow team-member"
                   src="images/Arjavi.jpg"
-                  alt=""
-                />
+                  alt=""/>
```

Comment:
Do we want to have no space (as seen here) between the ending tag and the closing /> brace or should we have one spacing (as seen in some of the code above e.g. the new line 8)?

Reply:
I think it is better to have one spacing. Thanks for pointing it out!